include/z64schedule.h

```diff
@@ -0,0 +1,248 @@
+    /* 18 */ SCHEDULE_CMD_ID_JUMP_L,             // Always jump, long range jump
+} ScheduleCommandID;
+
+typedef enum {
+    /* 0 */ SCHEDULE_ITEM_CHECK_ROOM_KEY,
+    /* 1 */ SCHEDULE_ITEM_CHECK_LETTER_TO_KAFEI,
+    /* 2 */ SCHEDULE_ITEM_CHECK_MASK_ROMANI,
+} ScheduleItemCheck;
+
+typedef struct {
+    /* 0x0 */ u8 result;
+    /* 0x4 */ s32 time0;
+    /* 0x8 */ s32 time1;
+    /* 0xC */ s32 hasResult;
+} ScheduleResult; // size = 0x10
+
+typedef struct {
+    /* 0x0 */ u8 cmd;
+} ScheduleCmdBase; // size = 0x1
+
+typedef struct {
+    /* 0x0 */ ScheduleCmdBase base;
+    /* 0x1 */ u8 flagByte;
+    /* 0x2 */ u8 flagMask;
+    /* 0x3 */ s8 offset;
+} ScheduleCmdFlagCheckS; // size = 0x4
+
+typedef struct {
+    /* 0x0 */ ScheduleCmdBase base;
+    /* 0x1 */ u8 flagByte;
+    /* 0x2 */ u8 flagMask;
+    /* 0x3 */ u8 offsetH;
+    /* 0x4 */ u8 offsetL;
+} ScheduleCmdFlagCheckL; // size = 0x5
+
+typedef struct {
+    /* 0x0 */ ScheduleCmdBase base;
+    /* 0x1 */ u8 startHr;
+    /* 0x2 */ u8 startMin;
+    /* 0x3 */ u8 endHr;
+    /* 0x4 */ u8 endMin;
+    /* 0x5 */ s8 offset;
+} ScheduleCmdTimeRangeCheckS; // size = 0x6
+
+typedef struct {
+    /* 0x0 */ ScheduleCmdBase base;
+    /* 0x1 */ u8 startHr;
+    /* 0x2 */ u8 startMin;
+    /* 0x3 */ u8 endHr;
+    /* 0x4 */ u8 endMin;
+    /* 0x5 */ u8 offsetH;
+    /* 0x6 */ u8 offsetL;
+} ScheduleCmdTimeRangeCheckL; // size = 0x7
+
+typedef struct {
+    /* 0x0 */ ScheduleCmdBase base;
+    /* 0x1 */ u8 retH;
+    /* 0x2 */ u8 retL;
+} ScheduleCmdReturnL; // size = 0x3
+
+typedef struct {
+    /* 0x0 */ ScheduleCmdBase base;
+    /* 0x1 */ u8 unk1;
+    /* 0x2 */ u8 unk2;
+    /* 0x3 */ u8 unk3;
+} ScheduleCmdNop; // size = 0x4
+
+typedef struct {
+    /* 0x0 */ ScheduleCmdBase base;
+    /* 0x1 */ u8 which;
+    /* 0x2 */ s8 offset;
+} ScheduleCmdItemCheckS; // size = 0x3
+
+typedef struct {
+    /* 0x0 */ ScheduleCmdBase base;
+    /* 0x1 */ u8 res; // TODO rename
+} ScheduleCmdReturnS; // size = 0x2
+
+typedef struct {
+    /* 0x0 */ ScheduleCmdBase base;
+    /* 0x1 */ u8 sceneH;
+    /* 0x2 */ u8 sceneL;
+    /* 0x3 */ s8 offset;
+} ScheduleCmdSceneCheckS; // size = 0x4
+
+typedef struct {
+    /* 0x0 */ ScheduleCmdBase base;
+    /* 0x1 */ u8 sceneH;
+    /* 0x2 */ u8 sceneL;
+    /* 0x3 */ u8 offsetH;
+    /* 0x4 */ u8 offsetL;
+} ScheduleCmdSceneCheckL; // size = 0x5
+
+typedef struct {
+    /* 0x0 */ ScheduleCmdBase base;
+    /* 0x1 */ u8 dayH;
+    /* 0x2 */ u8 dayL;
+    /* 0x3 */ s8 offset;
+} ScheduleCmdDayCheckS; // size = 0x4
+
+typedef struct {
+    /* 0x0 */ ScheduleCmdBase base;
+    /* 0x1 */ u8 dayH;
+    /* 0x2 */ u8 dayL;
+    /* 0x3 */ u8 offsetH;
+    /* 0x4 */ u8 offsetL;
+} ScheduleCmdDayCheckL; // size = 0x5
+
+typedef struct {
+    /* 0x0 */ ScheduleCmdBase base;
+    /* 0x1 */ u8 time0Hr;
+    /* 0x2 */ u8 time0Min;
+    /* 0x3 */ u8 time1Hr;
+    /* 0x4 */ u8 time1Min;
+    /* 0x5 */ u8 res; // TODO rename? (updadte macros too)
```

Comment:
Do you still plan on renaming this?

Reply:
Oh yeah I was going to standardize them all to `result`, whoops